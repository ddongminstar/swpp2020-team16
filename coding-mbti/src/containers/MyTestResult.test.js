import React from 'react';

import { shallow } from 'enzyme';
import MyTestResult from './MyTestResult';

describe('<MyTestResult/>', () => {
  let myTestResult;

  beforeEach(() => {
    myTestResult = <MyTestResult />;
  });

  it('should render withour any error', () => {
    const component = shallow(myTestResult);
    const wrapper = component.find('main');
    expect(wrapper.length).toBe(1);
  });
});
