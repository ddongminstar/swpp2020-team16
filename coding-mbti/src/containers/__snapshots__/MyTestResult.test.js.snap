// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyTestResult /> matches snapshot 1`] = `ShallowWrapper {}`;
